Formula/rust.rb

```diff
@@ -43,18 +43,18 @@ class Rust < Formula
     on_macos do
       # From https://github.com/rust-lang/rust/blob/#{version}/src/stage0.txt
       if Hardware::CPU.arm?
-        url "https://static.rust-lang.org/dist/2021-02-11/cargo-1.50.0-aarch64-apple-darwin.tar.gz"
-        sha256 "19d526ef3518fb0322f809deddbd4208a27d08efa41d2188348f1be8d3bcfe5e"
+        url "https://static.rust-lang.org/dist/2021-05-10/cargo-1.52.1-aarch64-apple-darwin.tar.gz"
+        sha256 "6787f8746cedd005b7abeabf6e2250ebc5fa767486c5bf23c597298e6965d576"
```

Comment:
This is incorrect: https://github.com/rust-lang/rust/blob/1.52.1/src/stage0.txt#L15-L16

Reply:
Thanks for your feedback! Not sure I understand the date part. Using the date `2021-03-25` in the url like this https://static.rust-lang.org/dist/2021-03-25/cargo-1.52.1-aarch64-apple-darwin.tar.gz results in a Not Found error for me